File: pysfml/sound.py
import os
from pysfml._core import CsSound, System

class Sound:
  def __init__(self, path):
    s_path = System.String(os.path.abspath(path))
    self._obj = System.Activator.CreateInstance(CsSound, s_path)

  def set_volume(self, v: float):
    self._obj.SetVolume(System.Single(v))
  
  def set_loop(self, is_looping: bool):
    self._obj.SetLoop(System.Boolean(is_looping))

  def play(self):
    self._obj.Play()

  def stop(self):
    self._obj.Stop()

  def pause(self):
    self._obj.Pause()

File: pysfmllib/Sound.cs
using SFML.Audio;

namespace pysfmllib
{
    public class Sound
    {
        private SFML.Audio.Sound sound;
        private SoundBuffer buffer;

        public Sound(string path)
        {
            buffer = new SoundBuffer(path);
            sound = new SFML.Audio.Sound(buffer);
        }

        public void Play() => sound.Play();
        public void Stop() => sound.Stop();
        public void Pause() => sound.Pause();
        public void SetLoop(bool IsLooping) => sound.IsLooping = IsLooping;
        public void SetVolume(float v) => sound.Volume = v;
    }
}
